tests/unittests/testFuselageStandardProfileSuperellipse.cpp

```diff
@@ -0,0 +1,174 @@
+        ReturnCode tixiRet;
+        TiglReturnCode tiglRet;
+
+        tiglHandle = -1;
+        tixiHandle = -1;
+
+        tixiRet = tixiOpenDocument(filename, &tixiHandle);
+        ASSERT_TRUE (tixiRet == SUCCESS);
+        tiglRet = tiglOpenCPACSConfiguration(tixiHandle, "", &tiglHandle);
+        ASSERT_TRUE(tiglRet == TIGL_SUCCESS);
+
+        // Test case on standardProfile rectangle with guide curves
+
+        const char* filename1 = "TestData/simpletest_standard_profile_superellipse_kink.cpacs.xml";
+        ReturnCode tixiRet1;
+        TiglReturnCode tiglRet1;
+
+        tiglHandle1 = -1;
+        tixiHandle1 = -1;
+
+        tixiRet1 = tixiOpenDocument(filename1, &tixiHandle1);
+        ASSERT_TRUE (tixiRet1 == SUCCESS);
+        tiglRet1 = tiglOpenCPACSConfiguration(tixiHandle1, "", &tiglHandle1);
+        ASSERT_TRUE(tiglRet1 == TIGL_SUCCESS);
+
+        // Test case on standardProfile, invalid elements
+
+        ReturnCode tixiRet2;
+        TiglReturnCode tiglRet2;
+
+        tiglHandle2 = -1;
+        tixiHandle2 = -1;
+
+        tixiRet2 = tixiOpenDocument(filename1, &tixiHandle2);
+        ASSERT_TRUE (tixiRet2 == SUCCESS);
+        tiglRet2 = tiglOpenCPACSConfiguration(tixiHandle2, "", &tiglHandle2);
+        ASSERT_TRUE(tiglRet2 == TIGL_SUCCESS);
+
+
+    }
+
+    static void TearDownTestCase()
+    {
+        ASSERT_TRUE(tiglCloseCPACSConfiguration(tiglHandle) == TIGL_SUCCESS);
+        ASSERT_TRUE(tixiCloseDocument(tixiHandle) == SUCCESS);
+        tiglHandle = -1;
+        tixiHandle = -1;
+
+        ASSERT_TRUE(tiglCloseCPACSConfiguration(tiglHandle1) == TIGL_SUCCESS);
+        ASSERT_TRUE(tixiCloseDocument(tixiHandle1) == SUCCESS);
+        tiglHandle1 = -1;
+        tixiHandle1 = -1;
+
+        ASSERT_TRUE(tiglCloseCPACSConfiguration(tiglHandle2) == TIGL_SUCCESS);
+        ASSERT_TRUE(tixiCloseDocument(tixiHandle2) == SUCCESS);
+        tiglHandle2 = -1;
+        tixiHandle2 = -1;
+
+    }
+
+    void SetUp() override {}
+    void TearDown() override {}
+
+
+    static TixiDocumentHandle           tixiHandle;
+    static TiglCPACSConfigurationHandle tiglHandle;
+
+    static TixiDocumentHandle           tixiHandle1;
+    static TiglCPACSConfigurationHandle tiglHandle1;
+
+
+    static TixiDocumentHandle           tixiHandle2;
+    static TiglCPACSConfigurationHandle tiglHandle2;
+};
+
+TixiDocumentHandle FuselageStandardProfileSuperEllipse::tixiHandle = 0;
+TiglCPACSConfigurationHandle FuselageStandardProfileSuperEllipse::tiglHandle = 0;
+TixiDocumentHandle FuselageStandardProfileSuperEllipse::tixiHandle1 = 0;
+TiglCPACSConfigurationHandle FuselageStandardProfileSuperEllipse::tiglHandle1 = 0;
+TixiDocumentHandle FuselageStandardProfileSuperEllipse::tixiHandle2 = 0;
+TiglCPACSConfigurationHandle FuselageStandardProfileSuperEllipse::tiglHandle2 = 0;
+
+TEST_F(FuselageStandardProfileSuperEllipse, BuildWireSuperEllipse)
```

Comment:
This test does not need the Test Fixture FuselageStandardProfileSuperEllipse, does it? Then I would prefer to use a normal test `TEST(XXX, YYY)` and maybe moving this to the test file for the common functions.

Reply:
Yes, it should be in the test file for the common functions, next to the other tests for profile wires. I moved it and updated the Assertions.